NMR Structural Profiling of Transcriptional Intermediates Reveals Riboswitch Regulation by Metastable RNA Conformations
#######################################################################################################################

:date: 2017-01-31
:modified: 2024-04-21
:tags: bacteria; energy landscapes; synthetic biology
:category: publications
:slug:
:author: mtw
:summary: We investigate the influence of co-transcriptional folding on ligand binding and conformational switching in a type I-A 2'-deoxyguanosine (2'dG)-sensing riboswitch from Mesoplasma florum
:title: Co-transcriptional folding as a key factor in riboswitch function


.. role:: link-flat-strong(link)
  :class: m-flat m-text m-strong

.. role:: link-flat(link)
  :class: m-flat m-text

.. role:: ul
  :class: m-text m-ul

.. role:: doi(link)
  :class: doi

Employing high-resolution NMR spectroscopy, this study unveils a tight interplay between RNA sequence, co-transcriptional folding dynamics, and ligand affinity. These findings establish a novel framework for dissecting the coupling between RNA synthesis and folding, emphasizing the role of metastable states in RNA-based gene regulation.


.. frame:: Abstract

  Gene repression induced by the formation of transcriptional terminators represents a prime example for the coupling of RNA synthesis, folding, and regulation. In this context, mapping the changes in available conformational space of transcription intermediates during RNA synthesis is important to understand riboswitch function. A majority of riboswitches, an important class of small metabolite-sensing regulatory RNAs, act as transcriptional regulators, but the dependence of ligand binding and the subsequent allosteric conformational switch on mRNA transcript length has not yet been investigated. We show a strict fine-tuning of binding and sequence-dependent alterations of conformational space by structural analysis of all relevant transcription intermediates at single-nucleotide resolution for the I-A type 2′dG-sensing riboswitch from Mesoplasma f lorum by NMR spectroscopy. Our results provide a general framework to dissect the coupling of synthesis and folding essential for riboswitch function, revealing the importance of metastable states for RNA-based gene regulation.



Citation
========

  | :link-flat-strong:`NMR Structural Profiling of Transcriptional Intermediates Reveals Riboswitch Regulation by Metastable RNA Conformations <https://doi.org/10.1021/jacs.6b10429>`
  | Christina Helmling, Anna Wacker, :ul:`Michael T. Wolfinger`, Ivo L. Hofacker, Martin Hengsbach, Boris Fürtig, Harald Schwalbe
  | *J. Am. Chem. Soc.* 139 (7):2647–56 (2017) | :doi:`doi:10.1021/jacs.6b10429 <https://doi.org/10.1021/jacs.6b10429>`


See also
========

  | :link-flat-strong:`Efficient Computation of Cotranscriptional RNA-Ligand Interaction Dynamics <{filename}/blog/2018-07-01-Efficient_Computation_of_Cotranscriptional_RNA-Ligand_Interaction_Dynamics.rst>`
  | :ul:`Michael T. Wolfinger`, Christoph Flamm, Ivo L. Hofacker
  | *Methods* 143:70–76 (2018) | :doi:`doi:10.1016/j.ymeth.2018.04.036 <https://doi.org/10.1016/j.ymeth.2018.04.036>` | :link-flat:`Preprint PDF <{static}/files/papers/Wolfinger-2018__PREPRINT.pdf>`
