Updated Phylogeny of Chikungunya Virus Suggests Lineage-Specific RNA Architecture
#################################################################################

:date: 2019-08-29
:modified: 2024-10-10
:tags: molecular epidemiology; One Health; virus bioinformatics; alphavirus; 
:category: publications
:slug: Updated-Phylogeny-of-Chikungunya-Virus-Suggests-Lineage-Specific-RNA-Architecture
:author: mtw
:title: Chikungunya Virus Phylogeny: Lineage-Specific RNA Structures
:description: Our study finds multiple independent clades in the ECSA lineage of Chikungunya virus and uncovers evidence of lineage-specific RNA structure conservation.
:summary: In this study, we analyzed the evolution and spread of the Chikungunya virus (CHIKV) across Africa by examining 598 whole genome sequences. Our analysis revealed several distinct CHIKV lineages with strong support, providing new insights into the virus's evolutionary history and its movement across regions

We carried out an in-depth analysis of the virus’s 3’UTR, where we uncovered conserved RNA structures that likely play crucial roles in stabilizing viral RNA and interacting with host proteins. One of our most intriguing findings was the discovery of unique 3’UTR structures specific to certain lineages. These features may help the virus counter host defenses and interact with key cellular proteins, such as the Musashi family of translational regulators.

Our comprehensive analysis provides valuable insights into the complex evolution and epidemiology of CHIKV. By highlighting lineage-specific traits, we hope this study will pave the way for a deeper understanding of viral genomics and open up new opportunities for targeted interventions. As we continue to explore the intricacies of CHIKV's genetic makeup, our findings represent an important step in the global effort to combat this significant public health threat.


.. role:: link-flat-strong(link)
  :class: m-flat m-text m-strong

.. role:: link-flat(link)
  :class: m-flat m-text

.. role:: ul
  :class: m-text m-ul

.. role:: doi(link)
  :class: doi

.. raw:: html

  <object data="{static}/files/papers/deBernardiSchneider-2019b.pdf" type="application/pdf" width="100%" height="1050px">
  <p>Your browser does not support PDFs. 
     <a href="{static}/files/papers/deBernardiSchneider-2019b.pdf">Download the PDF</a>
  </p>
  </object> <br/><br/>


.. frame:: Abstract

   Chikungunya virus (CHIKV), a mosquito-borne alphavirus of the family Togaviridae, has recently emerged in the Americas from lineages from two continents: Asia and Africa. Historically, CHIKV circulated as at least four lineages worldwide with both enzootic and epidemic transmission cycles. To understand the recent patterns of emergence and the current status of the CHIKV spread, updated analyses of the viral genetic data and metadata are needed. Here, we performed phylogenetic and comparative genomics screens of CHIKV genomes, taking advantage of the public availability of many recently sequenced isolates. Based on these new data and analyses, we derive a revised phylogeny from nucleotide sequences in coding regions. Using this phylogeny, we uncover the presence of several distinct lineages in Africa that were previously considered a single one. In parallel, we performed thermodynamic modeling of CHIKV untranslated regions (UTRs), which revealed evolutionarily conserved structured and unstructured RNA elements in the 3’UTR. We provide evidence for duplication events in recently emerged American isolates of the Asian CHIKV lineage and propose the existence of a flexible 3’UTR architecture among different CHIKV lineages.


Figures and Data
================

.. image-grid::

  {static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b/QuickSlide__deBernardiSchneider-2019b.001.png

  {static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b/QuickSlide__deBernardiSchneider-2019b.002.png
  {static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b/QuickSlide__deBernardiSchneider-2019b.003.png

  {static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b/QuickSlide__deBernardiSchneider-2019b.004.png
  {static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b/QuickSlide__deBernardiSchneider-2019b.005.png

  {static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b/QuickSlide__deBernardiSchneider-2019b.006.png
  {static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b/QuickSlide__deBernardiSchneider-2019b.007.png

Citation
========

  | :link-flat-strong:`Updated Phylogeny of Chikungunya Virus Suggests Lineage-Specific RNA Architecture <{filename}/blog/2019-08-29-Updated-Phylogeny-of-Chikungunya-Virus-Suggests-Lineage-Specific-RNA-Architecture.rst>`
  | Adriano de Bernardi Schneider, Roman Ochsenreiter, Reilly Hostager, Ivo L. Hofacker, Daniel Janies, :ul:`Michael T. Wolfinger`
  | *Viruses* 11:798 (2019) | :doi:`doi:10.3390/v11090798 <https://doi.org/10.3390/v11090798>` | :link-flat:`PDF <{static}/files/papers/deBernardiSchneider-2019b.pdf>` | :link-flat:`Figures <{static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b.pdf>`


See Also
========

  | :link-flat-strong:`Dynamic Molecular Epidemiology Reveals Lineage-Associated Single-Nucleotide Variants That Alter RNA Structure in Chikungunya Virus  <{filename}/blog/2021-02-08-Dynamic_Molecular_Epidemiology_Reveals_Lineage-Associated_Single-Nucleotide_Variants_That_Alter_RNA_Structure_in_Chikungunya_Virus.rst>`
  | Thomas Spicher, Markus Delitz, Adriano de Bernardi Schneider, :ul:`Michael T. Wolfinger`
  | *Genes* 12 (2):239 (2021) | :doi:`doi:10.3390/genes12020239 <https://doi.org/10.3390/genes12020239>` | :link-flat:`PDF <{static}/files/papers/Spicher-2021.pdf>` | :link-flat:`Figures <{static}/files/QuickSlide/QuickSlide__Spicher-2021.pdf>` 