Genomic Epidemiology of Superspreading Events in Austria Reveals Mutational Dynamics of SARS-CoV-2
##################################################################################################

:date: 2020-12-10
:modified: 2024-04-21
:tags: molecular epidemiology; One Health; virology
:category: publications
:slug:
:author: mtw
:summary: This study shows how superspreading events fueled the first wave of SARS-CoV-2 in Austria. We analyzed viral mutations and traced travel patterns, revealing the impact of superspreading on the course of the outbreak
:title: Genomic epidemiology of SARS-CoV-2 superspreading events in Austria
:description: Discover COVID-19 superspreading dynamics in Austria's first wave. Deep viral sequencing unveils SARS-CoV-2 spread and mutational dynamics.


.. role:: link-flat-strong(link)
  :class: m-flat m-text m-strong

.. role:: link-flat(link)
  :class: m-flat m-text

.. role:: ul
  :class: m-text m-ul

.. role:: doi(link)
  :class: doi


.. button-primary:: {static}/files/papers/Popa-2020.pdf

    Download PDF

.. frame:: Abstract

  Superspreading events shaped the coronavirus disease 2019 (COVID-19) pandemic, and their rapid identification and containment are essential for disease control. Here, we provide a national-scale analysis of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) superspreading during the first wave of infections in Austria, a country that played a major role in initial virus transmissions in Europe. Capitalizing on Austria’s well-developed epidemiological surveillance system, we identified major SARS-CoV-2 clusters during the first wave of infections and performed deep whole-genome sequencing of more than 500 virus samples. Phylogenetic-epidemiological analysis enabled the reconstruction of superspreading events and charts a map of tourism-related viral spread originating from Austria in spring 2020. Moreover, we exploited epidemiologically well-defined clusters to quantify SARS-CoV-2 mutational dynamics, including the observation of low-frequency mutations that progressed to fixation within the infection chain. Time-resolved virus sequencing unveiled viral mutation dynamics within individuals with COVID-19, and epidemiologically validated infector-infectee pairs enabled us to determine an average transmission bottleneck size of 103 SARS-CoV-2 particles. In conclusion, this study illustrates the power of combining epidemiological analysis with deep viral genome sequencing to unravel the spread of SARS-CoV-2 and to gain fundamental insights into mutational dynamics and transmission properties.



Citation
========

| :link-flat-strong:`Genomic Epidemiology of Superspreading Events in Austria Reveals Mutational Dynamics and Transmission Properties of SARS-CoV-2 <https://doi.org/10.1126/scitranslmed.abe2555>`
| Alexandra Popa, Jakob-Wendelin Genger, Michael D. Nicholson, Thomas Penz, Daniela Schmid, Stephan W Aberle, Benedikt Agerer, Alexander Lercher, Lukas Endler, Henrique Colaco, Mark Smyth, Michael Schuster, Miguel L. Grau, Francisco Martínez-Jiménez, Oriol Pich, Wegene Borena, Erich Pawelka, Zsofia Keszei, Martin Senekowitsch, Jan Laine, Judith H Aberle, Monika Redlberger-Fritz, Mario Karolyi, Alexander Zoufaly, Sabine Maritschnik, Martin Borkovec, Peter Hufnagl, Manfred Nairz, Günter Weiss, :ul:`Michael T. Wolfinger`, Dorothee von Laer, Giulio Superti-Furga, Nuria Lopez-Bigas, Elisabeth Puchhammer-Stöckl, Franz Allerberger, Franziska Michor, Christoph Bock, Andreas Bergthaler
| *Sci. Transl. Med.* 12 (573):eabe2555 (2020) | :doi:`doi:10.1126/scitranslmed.abe2555 <https://doi.org/10.1126/scitranslmed.abe2555>` | :link-flat:`PDF <{static}/files/papers/Popa-2020.pdf>`
