Dynamic Molecular Epidemiology Reveals Lineage-Associated Single-Nucleotide Variants That Alter RNA Structure in Chikungunya Virus
##################################################################################################################################

:date: 2021-02-08
:modified: 2022-10-28
:tags: virus bioinformatics; molecular epidemiology; alphavirus; virology
:category: publications
:slug: Lineage-specific-RNA-structures-in-Chikungunya-virus
:title: Molecular epidemiology and RNA structures in Chikungunya virus
:author: mtw
:summary: This research provides a comprehensive analysis of the molecular epidemiology and evolutionary dynamics of Chikungunya virus (CHIKV), uncovering how single nucleotide variants influence viral RNA structures and shape the global spread of the virus
:description: Dynamic Molecular Epidemiology Reveals Lineage-Associated Single-Nucleotide Variants That Alter RNA Structure in Chikungunya Virus

.. role:: link-flat-strong(link)
  :class: m-flat m-text m-strong

.. role:: link-flat(link)
  :class: m-flat m-text

.. role:: ul
  :class: m-text m-ul

.. role:: doi(link)
  :class: doi

.. container:: m-col-t-10 m-center-t m-col-s-10 m-center-s m-col-m-6 m-right-m

    .. figure:: {static}/files/papers/preview/Preview__Spicher-2021.001small.webp
          :alt: Enselble properties of a lineage-specific structured RNA in Chikungunya virus
          :figclass: m-figure m-flat


Chikungunya virus, an emerging Alphavirus, causes millions of human infections annually and has witnessed outbreaks in Africa and Asia since the 1950s. The research presented here delves into the molecular epidemiology of CHIKV, revealing how single nucleotide variants can impact the virus's RNA structures. Utilizing more than 1000 publicly available CHIKV genomes, the study provides an interactive phylodynamics dataset using Nextstrain, a tool for real-time tracking of pathogen evolution. The data set, publicly available through `CHIKVnext <https://nextstrain.org/groups/ViennaRNA/CHIKVnext>`_, enables the tracking of spatiotemporal and epidemiological aspects of the CHIKV global spread.

The study addresses genotype/phenotype associations in CHIKV by employing comparative approaches, molecular epidemiology concepts, phylogeny reconstruction, and computational RNA biology. The nucleotide divergence within CHIKV lineages is found to be relatively low, often due to geographical constraints and a limited collection period for newer lineages. However, the divergence between different lineages can be considerable, with the West African lineage showing the highest divergence compared to others.

This publication questions the current nomenclature of CHIKV lineages, and suggests that a new system, independent of geography, might be more coherent and beneficial for drug and vaccine development. Despite the low divergence in some established lineages, mutations like A226V in E1 can significantly impact the virus's capacity to replicate. The study further highlights the importance of increased surveillance to identify the virus at the time of introduction rather than during a pandemic.

This research exemplifies the importance of a holistic, interdisciplinary approach in understanding and combating infectious diseases such as CHIKV. The findings, which combine molecular epidemiology and RNA structure prediction, not only pave the way for better diagnostic and treatment methods but also underscore the necessity of a timely response to prevent outbreaks. The One Health approach is particularly pertinent here. By adopting a unified perspective, scientists, healthcare professionals, and policymakers can work together to address the multifaceted challenges posed by CHIKV and similar infectious diseases, ultimately contributing to the well-being of our global ecosystem.

.. raw:: html

  <object data="{static}/files/papers/Spicher-2021.pdf" type="application/pdf" width="100%" height="1050px">
  <p>Your browser does not support PDFs. 
     <a href="{static}/files/papers/Spicher-2021.pdf">Download the PDF</a>
  </p>
  </object> <br/><br/>

.. frame:: Abstract

  Chikungunya virus (CHIKV) is an emerging Alphavirus which causes millions of human infections every year. Outbreaks have been reported in Africa and Asia since the early 1950s, from three CHIKV lineages: West African, East Central South African, and Asian Urban. As new outbreaks occurred in the Americas, individual strains from the known lineages have evolved, creating new monophyletic groups that generated novel geographic-based lineages. Building on a recently updated phylogeny of CHIKV, we report here the availability of an interactive CHIKV phylodynamics dataset, which is based on more than 900 publicly available CHIKV genomes. We provide an interactive view of CHIKV molecular epidemiology built on Nextstrain, a web-based visualization framework for real-time tracking of pathogen evolution. CHIKV molecular epidemiology reveals single nucleotide variants that change the stability and fold of locally stable RNA structures. We propose alternative RNA structure formation in different CHIKV lineages by predicting more than a dozen RNA elements that are subject to perturbation of the structure ensemble upon variation of a single nucleotide.


Figures and Data
================

.. image-grid::

  {static}/files/QuickSlide/QuickSlide__Spicher-2021/QuickSlide__Spicher-2021.001.png

  {static}/files/QuickSlide/QuickSlide__Spicher-2021/QuickSlide__Spicher-2021.002.png
  {static}/files/QuickSlide/QuickSlide__Spicher-2021/QuickSlide__Spicher-2021.003.png

  {static}/files/QuickSlide/QuickSlide__Spicher-2021/QuickSlide__Spicher-2021.004.png
  {static}/files/QuickSlide/QuickSlide__Spicher-2021/QuickSlide__Spicher-2021.005.png

  {static}/files/QuickSlide/QuickSlide__Spicher-2021/QuickSlide__Spicher-2021.006.png
  {static}/files/QuickSlide/QuickSlide__Spicher-2021/QuickSlide__Spicher-2021.007.png

  {static}/files/QuickSlide/QuickSlide__Spicher-2021/QuickSlide__Spicher-2021.008.png



Citation
========

  | :link-flat-strong:`Dynamic Molecular Epidemiology Reveals Lineage-Associated Single-Nucleotide Variants That Alter RNA Structure in Chikungunya Virus  <{filename}/blog/2021-02-08-Dynamic_Molecular_Epidemiology_Reveals_Lineage-Associated_Single-Nucleotide_Variants_That_Alter_RNA_Structure_in_Chikungunya_Virus.rst>`
  | Thomas Spicher, Markus Delitz, Adriano de Bernardi Schneider, :ul:`Michael T. Wolfinger`
  | *Genes* 12 (2):239 (2021) | :doi:`doi:10.3390/genes12020239 <https://doi.org/10.3390/genes12020239>` | :link-flat:`PDF <{static}/files/papers/Spicher-2021.pdf>` | :link-flat:`Figures <{static}/files/QuickSlide/QuickSlide__Spicher-2021.pdf>` 

See Also
========

  | :link-flat-strong:`Updated Phylogeny of Chikungunya Virus Suggests Lineage-Specific RNA Architecture <{filename}/blog/2019-08-29-Updated-Phylogeny-of-Chikungunya-Virus-Suggests-Lineage-Specific-RNA-Architecture.rst>`
  | Adriano de Bernardi Schneider, Roman Ochsenreiter, Reilly Hostager, Ivo L. Hofacker, Daniel Janies, :ul:`Michael T. Wolfinger`
  | *Viruses* 11:798 (2019) | :doi:`doi:10.3390/v11090798 <https://doi.org/10.3390/v11090798>` | :link-flat:`PDF <{static}/files/papers/deBernardiSchneider-2019b.pdf>` | :link-flat:`Figures <{static}/files/QuickSlide/QuickSlide__deBernardiSchneider-2019b.pdf>`
