Bi-Alignments as Models of Incongruent Evolution of RNA Sequence and Secondary Structure
########################################################################################
:summary: Bi-Alignments as Models of Incongruent Evolution of RNA Sequence and Secondary Structure


:breadcrumb: / Home
             /publications/ Papers

.. role:: ul
 :class: m-text m-ul

.. role:: doi(link)
 :class: doi

Maria Waldl, Sebastian Will, :ul:`Michael T. Wolfinger`, Ivo L. Hofacker, Peter F. Stadler

In *Computational Intelligence Methods for Bioinformatics and Biostatistics*, pp159â€“70. Springer International Publishing (2020)

doi: :doi:`10.1007/978-3-030-63061-4_15 <https://doi.org/10.1007/978-3-030-63061-4_15>`

:link-flat-strong:`Preprint PDF <{static}/files/papers/Waldl-2020__PREPRINT.pdf>`

Abstract
========

RNA molecules may be subject to independent selection pressures on sequence and structure. This can, in principle, lead to the preservation of structural features without maintaining the exact position on the conserved sequence. Consequently, structurally analogous base pairs are no longer formed by homologous bases, and homologous nucleotides do not preserve their structural context. In other words, the evolution of sequence and structure is incongruent. We model this phenomenon by introducing bi-alignments, defined as a pair of alignments, one modeling sequence homology; the other, structural homology, together with an alignment of the two alignments that models the relative shifts between conserved sequence and conserved structure. Bi-alignments therefore form a special class of four-way alignments. A preliminary survey of the Rfam database suggests that incongruent evolution is not a very rare phenomenon among structured ncRNAs and RNA elements.
