<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
  <meta charset="utf-8">
  <title>ViennaNGS Installation HOWTO - rna-seq.at</title>
  <meta name="author" content="Michael T. Wolfinger">

  
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://mtw.github.io/blog/2015/03/03/viennangs-installation-howto/">
  <link href="/favicon.png" type="image/png" rel="icon">
  <link href="/atom.xml" rel="alternate" title="rna-seq.at" type="application/atom+xml">

  <!-- http://opengraphprotocol.org/ -->
  <meta name="twitter:card" content="summary_large_image">
  <meta property="og:type" content="website">
  <meta property="og:url" content="http://mtw.github.io/blog/2015/03/03/viennangs-installation-howto/">
  <meta property="og:title" content="ViennaNGS Installation HOWTO - rna-seq.at">
  

  <script src="/javascripts/libs/jquery/jquery-2.1.3.min.js"></script>

<link href="/assets/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css">
<link href="/assets/bootstrap/dist/css/bootstrap-theme.min.css" rel="stylesheet" type="text/css">


  
  <link href="/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css">

  
   <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-15287216-3', 'auto');
    ga('send', 'pageview');

  </script>


</head>

  <body   >
    <a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>
    <div id="wrap">
      <header role="banner">
        <nav class="navbar navbar-default" role="navigation">
    <div class="container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" title="toggle navbar" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">rna-seq.at</a>
        </div>

        <div class="navbar-collapse collapse">
            <ul class="nav navbar-nav">
                <li class="active">
                    <a rel="index" href="/">Blog</a>
                </li>
                <li >
                    <a href="/blog/archives">Archives</a>
                </li>
                <li >
                    <a href="/about">About</a>
                </li>
            </ul>
            <ul class="nav navbar-nav navbar-right">
                <li>
                    <a class="subscribe-rss" href="/atom.xml" title="subscribe via RSS">
                        <span class="visible-xs">RSS</span>
                        <img class="hidden-xs" src="/images/rss.png" alt="RSS">
                    </a>
                </li>
                
            </ul>
            
                <form class="navbar-form navbar-right" action="https://www.google.com/search" method="GET">
                    <input type="hidden" name="sitesearch" value="mtw.github.io">
                    <div class="form-group">
                        <input class="form-control" type="text" name="q" placeholder="Search">
                    </div>
                </form>
            
        </div>
    </div>
</nav>


      </header>
      <div id="main" role="main" class="container">
        <div id="content">
          <div class="row">
  <div class="page-content col-md-9" itemscope itemtype="http://schema.org/Blog">
    <meta itemprop="name" content="rna-seq.at" />
    <meta itemprop="description" content="Stuff that matters in open source RNA bioinformatics" />
    <meta itemprop="url" content="http://mtw.github.io" />
    <article class="hentry" role="article" itemprop="blogPost" itemscope itemtype="http://schema.org/BlogPosting">
      
  <header class="page-header">
    
      <p class="meta text-muted text-uppercase">
        












<span class="glyphicon glyphicon-calendar"></span> <time datetime="2015-03-03T11:45:50+01:00"  data-updated="true" itemprop="datePublished dateCreated">2015-03-03</time>
        
      </p>
    
    
    <h1 class="entry-title" itemprop="name headline">
        ViennaNGS Installation HOWTO
        
    </h1>
    
  </header>


<div class="entry-content clearfix" itemprop="articleBody description"><p><a href="http://search.cpan.org/dist/Bio-ViennaNGS">ViennaNGS</a> is a Perl distribution
for rapid development of next-generation sequencing analysis pipelines. It comes
with a set of modules and <a href="http://moose.iinteractive.com/en/">Moose-based</a>
classes for accomplishing standard and non-standard tasks in NGS processing.
ViennaNGS' feature-richness, however, comes at some cost: dependencies on third
party Perl modules and external tools and libraries, which can be tedious to
install. I have therefore compiled this little <strong>ViennaNGS Installation HOWTO</strong>
to help prospective users getting the software up and running quickly on their
systems.</p>

<p>Let&rsquo;s get started. The installation process provided here has been prepared for
Linux systems, however the steps should be reproducible on MacOS X given that
you have the common GNU tools and  Perl 5.10.0 or higher available, e.g. via
<a href="https://www.macports.org">MacPorts</a>.</p>

<h3>First things first: third party dependencies</h3>

<p><em>ViennaNGS</em> depends on set of third party bioinf tools and libraries, which are
required either for specific filtering and file format conversion tasks or for
building internally used Perl modules. Before continuing with the <em>ViennaNGS</em>
installation, you will need to download and install the following software and
ensure that all executables are accessible to the Perl interpreter:</p>

<ul>
<li>A recent version of <a href="https://github.com/arq5x/bedtools2">bedtools2</a> (latest version as of writing was v2.23)</li>
<li>The <em>bedGraphToBigWig</em> and <em>fetchChromSizes</em> <a href="http://hgdownload.cse.ucsc.edu/admin/exe/">UCSC Genome Browser application binaries</a></li>
<li>The <a href="http://www.r-project.org/">R Statistics Package</a></li>
<li><a href="https://github.com/samtools/samtools/releases/tag/0.1.19">samtools &lt;= v0.1.19</a>. More recent HTSlib-based versions will <strong>not work</strong> with <a href="http://search.cpan.org/~lds/Bio-SamTools/lib/Bio/DB/Sam.pm">Bio::DB::Sam</a></li>
</ul>


<p>The last dependency is in fact a bit peculiar. <em>Bio::DB::Sam</em> up to version
1.41, which we use for all kind of direct BAM manipulation within <em>ViennaNGS</em>,
for some reason will not compile with recent, HTSlib-based versions of
<em>samtools</em>. <em>samtools 0.1.19</em> seems to be the last version that is compatible
with current versions of <em>Bio::DB::Sam</em>. So let&rsquo;s download <em>samtools 0.1.19</em> to
a local folder, say <code>/scratch/software</code>, uncompress the tarball and compile the
samtools library:</p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
<span class='line-number'>2</span>
<span class='line-number'>3</span>
<span class='line-number'>4</span>
<span class='line-number'>5</span>
<span class='line-number'>6</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>mkdir -p /scratch/software
</span><span class='line'>cd /scratch/software
</span><span class='line'>wget https://github.com/samtools/samtools/archive/0.1.19.tar.gz
</span><span class='line'>tar xvf 0.1.19.tar.gz
</span><span class='line'>cd samtools-0.1.19
</span><span class='line'>make CFLAGS="-g -O2 -Wall -Wno-unused -Wno-unused-result -fPIC"</span></code></pre></td></tr></table></div></figure>


<p>The extra <code>-Wno*</code> CFLAGS options prevent some unused variable warnings with
recent versions of <em>gcc</em>, whereas the <code>-fPIC</code> CFLAGS option prohibits a
relocation error on x64_64 architectures (see also the <a href="http://cpansearch.perl.org/src/LDS/Bio-SamTools-1.41/README">Bio::DB::Sam
README</a>). If
everything works out fine,  <em>samtools</em> is now compiled, and a static library
file <code>libbam.a</code> will be created in the current working directory. Remember the
absolute path to <code>libbabm.a</code> (in our example this is
<code>/scratch/software/samtools-0.1.19</code>), since we will need this later for
compiling <em>Bio::DB::Sam</em>.</p>

<h3>Setting up a local Perl module directory with cpanminus</h3>

<p>Once all thrid-party dependencies are installed, we can now focus on the Perl
dependencies of <em>ViennaNGS</em>, e.g. the <em>BioPerl</em> suite, the <em>Moose</em> object
framework and many more. Fortunately, installation of custom Perl modules (i.e.
those that are not considered Perl core modules or shipped with a Linux
distribution) is a fairly trivial task these days, thanks to a tool called
<a href="https://cpanmin.us">cpanminus</a>. <em>cpanminus</em> renders installation of a custom
Perl module as easy as typing <code>cpanm</code> and the module&rsquo;s name on the command line.
Moreover, it provides automatic dependency resolution, which comes in handy when
you&rsquo;re installing Perl modules or distributions that depend on other modules,
which again depend on a bunch of modules etc. Forget about the <em>CPAN shell</em> or
<em>cpanplus</em>, use <em>cpanminus</em>.</p>

<p>Before starting with <em>cpanminus</em> we&rsquo;ll need to set a few environment variables
that tell <em>cpanm</em> and the Perl interpreter where to keep and look for locally
installed Perl modules. We&rsquo;ll keep up with the <em>cpanminus</em> default location,
<code>${HOME}/perl5</code> and add the following lines to the end of <code>.bashrc</code>:</p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
<span class='line-number'>2</span>
<span class='line-number'>3</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>export PERL_MM_OPT="INSTALL_BASE=${HOME}/perl5"
</span><span class='line'>export PERL_MB_OPT="--install_base ${HOME}/perl5"
</span><span class='line'>export PERL5LIB=${HOME}/perl5/lib/perl5:${PERL5LIB}</span></code></pre></td></tr></table></div></figure>


<p>We will also modify our <code>PATH</code> variable to include <code>${HOME}/perl5/bin</code>, where
executable scripts that are shipped with custom Perl modules will be installed:</p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>export PATH=${HOME}/perl5/bin:${PATH}</span></code></pre></td></tr></table></div></figure>


<p>We can now install <em>cpanminus</em>, either the version shipped with our Linux
distribution (example given for Debian/Ubuntu must be executed with root
privileges)</p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>apt-get install cpanminus</span></code></pre></td></tr></table></div></figure>


<p>or directly from <a href="https://cpanmin.us">https://cpanmin.us</a></p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>curl -L https://cpanmin.us | perl - App::cpanminus</span></code></pre></td></tr></table></div></figure>


<h3>Installing the ViennaNGS Perl dependencies</h3>

<p>After all we are now ready to install the <em>ViennaNGS</em> Perl dependencies. Let&rsquo;s start with <em>Bio::DB::Sam</em></p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>cpanm Bio::DB::Sam</span></code></pre></td></tr></table></div></figure>


<p>Depending on your system setup, <em>cpanm</em> will now start to download and install
all dependencies of <em>Bio::DB::Sam</em> and will then ask for the location of the
static <code>libbam.a</code> library:</p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>Configuring Bio-SamTools-1.41 ... Please enter the location of the bam.h and compiled libbam.a files:</span></code></pre></td></tr></table></div></figure>


<p>Our <em>samtools</em> folder is <code>/scratch/software/samtools-0.1.19</code> (see above), so let&rsquo;s enter this here:</p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>/scratch/software/samtools-0.1.19</span></code></pre></td></tr></table></div></figure>


<p>This was basically the only module that we had to install manually due to the
<em>samtools</em> location constraint. Actually, we could have installed
<em>Bio::ViennaNGS</em> directly and <em>cpanm</em> would have asked for the <em>samtools</em>
location, but I thought this was a good chance to demonstrate <em>cpanm</em> usage for
didactic reasons.</p>

<h3>Installing Bio::ViennaNGS</h3>

<p>Here comes the easiest part. Since we&rsquo;ve got <code>cpanm</code> up and running on our
system, all we need to type is</p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>cpanm Bio::ViennaNGS</span></code></pre></td></tr></table></div></figure>


<p>and wait for <em>cpanm</em> to install all dependencies and finish. Beware that
<em>ViennaNGS</em> depends on
<a href="http://search.cpan.org/~fangly/Statistics-R/lib/Statistics/R.pm">Statistics::R</a>,
which requires a running installation of the <a href="http://www.r-project.org/">R Statistics
Package</a> on your system. If for any reason <em>cpanm</em>
fails (e.g. R is not available), it will stop and report an error. You can then
install the missing component and call the above command again. Moreover,
whenever there is a new <em>ViennaNGS</em> release available on CPAN, just type <code>cpanm
Bio::ViennaNGS</code> and you&rsquo;ll get the latest version in just a couple of seconds.</p>

<p>Congratulations! You now have a running installation of <em>Bio::ViennaNGS</em>. If you
sticked with the paths in our example, the modules/classes are located in
<code>${HOME}/perl5/lib/perl5/Bio/ViennaNGS/lib</code> and the <em>ViennaNGS utilities</em> can be
found in <code>${HOME}/perl5/bin</code>.</p>

<p>The <em>ViennaNGS</em> authors are looking forward to getting your feedback. Please use
the <a href="https://github.com/mtw/Bio-ViennaNGS/issues">ViennaNGS GitHub Issue
Tracker</a> for reporting bugs.</p>
</div>


      <footer>
        <p class="meta text-muted">
          
  

<span class="glyphicon glyphicon-user"></span> <span class="byline author vcard" itemprop="author" itemscope itemtype="http://schema.org/Person">Posted by <span class="fn" itemprop="name">Michael T. Wolfinger</span></span>

          












<span class="glyphicon glyphicon-calendar"></span> <time datetime="2015-03-03T11:45:50+01:00"  data-updated="true" itemprop="datePublished dateCreated">2015-03-03</time>
          

<span class="glyphicon glyphicon-tags"></span>&nbsp;
<span class="categories">
  
    <a class='category' href='/blog/categories/viennangs/'>viennangs</a>
  
</span>


        </p>
        
          <div class="sharing">
  
  
  
    <div class="fb-like" data-send="true" data-width="450" data-show-faces="false"></div>
  
</div>

        
        
          <ul class="meta text-muted pager">
            
            <li class="previous"><a href="/blog/2015/03/02/viennangs-a-toolbox-for-building-efficient-next-generation-sequencing-analysis-pipelines/" title="Previous Post: ViennaNGS: A toolbox for building efficient next-generation sequencing analysis pipelines">&laquo; ViennaNGS: A toolbox for building efficient next-generation sequencing analysis pipelines</a></li>
            
            
          </ul>
        
      </footer>
    </article>
    
  </div>

  
  <aside class="sidebar col-md-3">
    
      <section class="panel panel-default">
  <div class="panel-heading">
    <h3 class="panel-title">Recent Posts</h3>
  </div>
  
  <div id="recent_posts" class="list-group">
    
    <a class="list-group-item active" href="/blog/2015/03/03/viennangs-installation-howto/">ViennaNGS Installation HOWTO</a>
    
    <a class="list-group-item " href="/blog/2015/03/02/viennangs-a-toolbox-for-building-efficient-next-generation-sequencing-analysis-pipelines/">ViennaNGS: A Toolbox for Building Efficient Next-generation Sequencing Analysis Pipelines</a>
    
    <a class="list-group-item " href="/blog/2014/04/15/how-to-compute-normalized-rna-seq-expression-from-multicov-files/">How to Compute Normalized RNA-seq Expression From Multicov Files</a>
    
    <a class="list-group-item " href="/blog/2014/04/13/tssar-tss-annotation-regime-for-drna-seq-data/">TSSAR: TSS Annotation Regime for dRNA-seq Data</a>
    
    <a class="list-group-item " href="/blog/2013/12/18/tivoli-storage-manager-under-debian-7-wheezy/">Tivoli Storage Manager Under Debian 7 Wheezy</a>
    
  </div>
</section>

<section class="twitterOct">
  <h1 style="margin-bottom: 0.4em"> Tweets </h1>
  <a class="twitter-timeline"
	 data-dnt="true" href="https://twitter.com/mtwolfinger" 
     data-widget-id="412878550417420288">
     
     Tweets by @mtwolfinger
  </a>
  <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
</section>


    
  </aside>
  
</div>

        </div>
      </div>
    </div>
    <footer role="contentinfo"><div class="container">
    <p class="text-muted credits">
  Copyright &copy; 2015 - Michael T. Wolfinger<br>
  <small>
      <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>,
      <span class="credit">customized with <a href="https://github.com/kAworu/octostrap3">octostrap3</a></span>.
  </small>
</p>

</div>
</footer>
    



<div id="fb-root"></div>
<script>(function(d, s, id) {
  var js, fjs = d.getElementsByTagName(s)[0];
  if (d.getElementById(id)) {return;}
  js = d.createElement(s); js.id = id; js.async = true;
  js.src = "//connect.facebook.net/en_US/all.js#appId=212934732101925&xfbml=1";
  fjs.parentNode.insertBefore(js, fjs);
}(document, 'script', 'facebook-jssdk'));</script>






<script src="/assets/bootstrap/dist/js/bootstrap.min.js"></script>
<script src="/javascripts/modernizr.js"></script>


  </body>
</html>
