<!DOCTYPE html>
<html lang="en" prefix="og: http://ogp.me/ns#">
<head>
  <meta charset="UTF-8" />
  <title>TSSAR: Bacterial transcription start site annotation from dRNA-seq data</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300;400|Source+Code+Pro:200:300,400|Source+Sans+Pro:300,400" />
  <link rel="stylesheet" href="https://michaelwolfinger.com/static/m-light.compiled.min.css" />
  <link rel="stylesheet" href="https://michaelwolfinger.com/static/m-mtw.min.css" />
  <link rel="canonical" href="https://michaelwolfinger.com/blog/2014/TSSAR-tss-annotation-regime-for-drna-seq-data/" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <meta name="theme-color" content="#EAEAEA" />
  <meta name="description" content="TSSAR: A RESTful web service for automated de-novo annotation of bacterial transcription start sites from differential RNA-sequencing (dRNA-seq) data" />
  <meta name="twitter:site" content="@mtwolfinger" />
  <meta name="twitter:site:id" content="15105886" />
  <meta property="og:site_name" content="michaelwolfinger.com Bioinformatics" />
  <meta property="og:title" content="TSSAR: Bacterial transcription start site annotation from dRNA-seq data" />
  <meta name="twitter:title" content="TSSAR: Bacterial transcription start site annotation from dRNA-seq data" />
  <meta property="og:url" content="https://michaelwolfinger.com/blog/2014/TSSAR-tss-annotation-regime-for-drna-seq-data/" />
  <meta property="og:description" content="A RESTful web service and standalone application for automated de-novo annotation of bacterial transcription start sites from dRNA-seq data" />
  <meta name="twitter:description" content="A RESTful web service and standalone application for automated de-novo annotation of bacterial transcription start sites from dRNA-seq data" />
  <meta name="twitter:card" content="summary" />
  <meta property="og:type" content="article" />
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-T2RW3VW55N"></script>
  <script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-T2RW3VW55N');
  </script>
</head>
<body>
<header><nav id="navigation">
  <div class="m-container">
    <div class="m-row">
      <a href="../../../" id="m-navbar-brand" class="m-col-t-9 m-col-m-none m-left-m">Michael T. Wolfinger Bioinformatics</a>
      <a id="m-navbar-show" href="#navigation" title="Show navigation" class="m-col-t-3 m-hide-m m-text-right"></a>
      <a id="m-navbar-hide" href="#" title="Hide navigation" class="m-col-t-3 m-hide-m m-text-right"></a>
      <div id="m-navbar-collapse" class="m-col-t-12 m-show-m m-col-m-none m-right-m">
        <div class="m-row">
          <ol class="m-col-t-12 m-col-m-none">
            <li><a href="https://michaelwolfinger.com/consulting">Consulting</a></li>
            <li>
              <a href="https://michaelwolfinger.com/publications">Publications</a>
              <ol>
                <li><a href="https://michaelwolfinger.com/publications">Papers</a></li>
                <li><a href="https://michaelwolfinger.com/publications/presentations">Presentations</a></li>
                <li><a href="https://michaelwolfinger.com/publications/posters">Posters</a></li>
              </ol>
            </li>
            <li><a href="https://michaelwolfinger.com/teaching">Teaching</a></li>
            <li><a href="https://michaelwolfinger.com/contact">Contact</a></li>
            <li><a href="https://michaelwolfinger.com/blog/" id="m-navbar-current">Blog</a></li>
          </ol>
        </div>
      </div>
    </div>
  </div>
</nav></header>
<main>
<div class="m-container">
  <div class="m-row">
    <article class="m-col-m-10 m-nopadb">
      <header>
        <h1><a href="https://michaelwolfinger.com/blog/2014/TSSAR-tss-annotation-regime-for-drna-seq-data/" rel="bookmark" title="Permalink to TSSAR: Bacterial transcription start site annotation from dRNA-seq data">
          <time class="m-date" datetime="2014-04-13T00:00:00+02:00">
            Apr <span class="m-date-day">13</span> 2014
          </time>
          TSSAR: Bacterial transcription start site annotation from dRNA-seq data
        </a></h1>
        <p>A RESTful web service and standalone application for automated de-novo annotation of bacterial transcription start sites from dRNA-seq data</p>
      </header>
      <div class="m-clearfix-l"></div>
<!-- content -->
<p><em>In silico</em> identification of bacterial transcription start sites (TSS) has
been a major challenge for the last years. To address this issue, we have
developped <a href="http://rna.tbi.univie.ac.at/TSSAR">TSSAR</a>, a statistical method
for analyzing dRNA-seq data, together with colleagues from the
Bioinformatics department at the University of Leipzig.</p>
<div class="m-button m-primary">
<a href="../../../files/papers/Amman-2014.pdf"><span>Download PDF</span></a>
</div>
<aside class="m-frame">
<h3>Abstract</h3>
<p><strong>Background:</strong> Differential RNA sequencing dRNA-seq is a high-throughput screening
technique designed to examine the architecture of bacterial operons in
general and the precise position of transcription start sites (TSS) in
particular. Hitherto, dRNA-seq data were analyzed by visualizing the
sequencing reads mapped to the reference genome and manually annotating
reliable positions. This is very labor intensive and, due to the
subjectivity, biased.</p>
<p><strong>Results:</strong> Here, we present <strong>TSSAR</strong>, a tool for automated de-novo TSS annotation
from dRNA-seq data that respects the statistics of dRNA-seq
libraries. <strong>TSSAR</strong> uses the premise that the number of sequencing reads
starting at a certain genomic position within a transcriptional active
region follows a Poisson distribution with a parameter that depends on the
local strength of expression. The differences of two dRNA-seq library
counts thus follow a Skellam distribution. This provides a statistical
basis to identify significantly enriched primary transcripts.</p>
<p>We assessed the performance by analyzing a publicly available dRNA-seq
data set using <strong>TSSAR</strong> and two simple approaches that utilize
user-defined score cutoffs. We evaluated the power of reproducing the
manual TSS annotation. Furthermore, the same data set was used to
reproduce 74 experimentally validated TSS in <em>H. pylori</em> from reliable
techniques such as RACE or primer extension. Both analyses showed that
<strong>TSSAR</strong> outperforms the static cutoff-dependent approaches.</p>
<p><strong>Conclusions:</strong> Having an automated and efficient tool for analyzing dRNA-seq data
facilitates the use of the dRNA-seq technique and promotes its
application to more sophisticated analysis. For instance, monitoring the
plasticity and dynamics of the transcriptomal architecture triggered by
different stimuli and growth conditions becomes possible.</p>
<p>The main asset of a novel tool for dRNA-seq analysis that reaches out to
a broad user community is usability. As such, we provide <strong>TSSAR</strong> both as
intuitive RESTful Web service <a class="m-link-wrap" href="http://rna.tbi.univie.ac.at/TSSAR">http://rna.tbi.univie.ac.at/TSSAR</a> together
with a set of post-processing and analysis tools, as well as a
stand-alone version for use in high-throughput dRNA-seq data analysis
pipelines.</p>
</aside>
<section id="citation">
<h2>Citation</h2>
<blockquote>
<p>
<a class="m-flat m-text m-strong" href="http://www.biomedcentral.com/1471-2105/15/89">TSSAR: TSS annotation regime for dRNA-seq data</a><br />
Fabian Amman, Michael T. Wolfinger, Ronny Lorenz, Ivo L. Hofacker, Peter F. Stadler, Sven Findeiß<br />
<em>BMC Bioinformatics</em> 15:89 (2014) | <a class="doi" href="https://doi.org/10.1186/1471-2105-15-89">doi: 10.1186/1471-2105-15-89</a> | <a class="m-flat m-text" href="../../../files/papers/Amman-2014.pdf">PDF</a><br />
</p>
</blockquote>
</section>
<!-- /content -->
      <footer>
        <p>Posted by Michael T. Wolfinger on <time datetime="2014-04-13T00:00:00+02:00">Sun 13 April 2014</time>. <span class="m-label m-success">updated <time datetime="2022-09-27T00:00:00+02:00">Tue 27 September 2022</time></span>
      </footer>
    </article>
    <nav class="m-navpanel m-col-m-2">
      <h3>Categories</h3>
      <ol class="m-block-bar-m">
        <li><a href="https://michaelwolfinger.com/blog/category/howto/">HOWTO</a></li>
        <li><a href="https://michaelwolfinger.com/blog/category/publications/">publications</a></li>
      </ol>
      <h3>Tag cloud</h3>
      <ul class="m-tagcloud">
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/3d/">3D</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/bacteria/">bacteria</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/energy-landscapes/">energy landscapes</a></li>
        <li class="m-tag-5"><a href="https://michaelwolfinger.com/blog/tag/ngs/">NGS</a></li>
        <li class="m-tag-5"><a href="https://michaelwolfinger.com/blog/tag/non-coding-rna/">non-coding RNA</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/rna-design/">RNA design</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/rna-kinetics/">RNA kinetics</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/shape/">SHAPE</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/viennarna/">ViennaRNA</a></li>
        <li class="m-tag-5"><a href="https://michaelwolfinger.com/blog/tag/viruses/">viruses</a></li>
      </ul>
    </nav>
  </div>
</div>
</main>
<footer><nav>
  <div class="m-container">
    <div class="m-row">
      <div class="m-col-s-3 m-col-t-6">
        <h3><a href="https://michaelwolfinger.com/">Home</a></h3>
        <ul>
          <li><a href="https://michaelwolfinger.com/publications">Publications</a></li>
          <li><a href="https://michaelwolfinger.com/teaching">Teaching</a></li>
          <li><a href="https://michaelwolfinger.com/contact">Contact</a></li>
          <li><a href="https://michaelwolfinger.com/blog">Blog</a></li>
        </ul>
      </div>
      <div class="m-col-s-3 m-col-t-6">
        <h3>Services</h3>
        <ul>
          <li><a href="https://michaelwolfinger.com/consulting">Bioinformatics Consulting</a></li>
          <li><a href="https://michaelwolfinger.com/">Training</a></li>
        </ul>
      </div>
      <div class="m-clearfix-t"></div>
      <div class="m-col-s-3 m-col-t-6">
        <h3>My Profiles</h3>
        <ul>
          <li><a href="https://scholar.google.at/citations?user=w0PHGnEAAAAJ&amp;hl=en">Google Scholar</a></li>
          <li><a href="https://www.webofscience.com/wos/author/record/N-9538-2014">Web of Science</a></li>
          <li><a href="https://www.researchgate.net/profile/Michael-Wolfinger">ResearchGate</a></li>
          <li><a href="https://www.scopus.com/authid/detail.uri?authorId=6508361997">Scopus</a></li>
        </ul>
      </div>
      <div class="m-col-s-3 m-col-t-6">
        <h3>Social</h3>
        <ul>
          <li><a href="https://www.linkedin.com/in/michaelwolfinger/">Linkedin</a></li>
          <li><a href="https://twitter.com/mtwolfinger">Twitter</a></li>
          <li><a href="https://github.com/mtw">GitHub</a></li>
          <li><a href="https://orcid.org/0000-0003-0925-5205">ORCID</a></li>
        </ul>
      </div>
    </div>
    <div class="m-row">
      <div class="m-col-l-10 m-push-l-1">
        <p>
        michaelwolfinger.com - Copyright © Michael T. Wolfinger, 2021-2023<br />
        </p>
      </div>
    </div>
  </div>
</nav></footer>
</body>
</html>