<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
  <meta charset="utf-8">
  <title>TSSAR: TSS Annotation Regime for dRNA-seq Data - rna-seq.at</title>
  <meta name="author" content="Michael T. Wolfinger">

  
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://mtw.github.io/blog/2014/04/13/tssar-tss-annotation-regime-for-drna-seq-data/">
  <link href="/favicon.png" type="image/png" rel="icon">
  <link href="/atom.xml" rel="alternate" title="rna-seq.at" type="application/atom+xml">

  <!-- http://opengraphprotocol.org/ -->
  <meta name="twitter:card" content="summary_large_image">
  <meta property="og:type" content="website">
  <meta property="og:url" content="http://mtw.github.io/blog/2014/04/13/tssar-tss-annotation-regime-for-drna-seq-data/">
  <meta property="og:title" content="TSSAR: TSS Annotation Regime for dRNA-seq Data - rna-seq.at">
  

  <script src="/javascripts/libs/jquery/jquery-2.1.3.min.js"></script>

<link href="/assets/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css">
<link href="/assets/bootstrap/dist/css/bootstrap-theme.min.css" rel="stylesheet" type="text/css">


  
  <link href="/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css">

  
   <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-15287216-3', 'auto');
    ga('send', 'pageview');

  </script>


</head>

  <body   >
    <a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>
    <div id="wrap">
      <header role="banner">
        <nav class="navbar navbar-default" role="navigation">
    <div class="container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" title="toggle navbar" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">rna-seq.at</a>
        </div>

        <div class="navbar-collapse collapse">
            <ul class="nav navbar-nav">
                <li class="active">
                    <a rel="index" href="/">Blog</a>
                </li>
                <li >
                    <a href="/blog/archives">Archives</a>
                </li>
                <li >
                    <a href="/about">About</a>
                </li>
            </ul>
            <ul class="nav navbar-nav navbar-right">
                <li>
                    <a class="subscribe-rss" href="/atom.xml" title="subscribe via RSS">
                        <span class="visible-xs">RSS</span>
                        <img class="hidden-xs" src="/images/rss.png" alt="RSS">
                    </a>
                </li>
                
            </ul>
            
                <form class="navbar-form navbar-right" action="https://www.google.com/search" method="GET">
                    <input type="hidden" name="sitesearch" value="mtw.github.io">
                    <div class="form-group">
                        <input class="form-control" type="text" name="q" placeholder="Search">
                    </div>
                </form>
            
        </div>
    </div>
</nav>


      </header>
      <div id="main" role="main" class="container">
        <div id="content">
          <div class="row">
  <div class="page-content col-md-9" itemscope itemtype="http://schema.org/Blog">
    <meta itemprop="name" content="rna-seq.at" />
    <meta itemprop="description" content="Stuff that matters in open source RNA bioinformatics" />
    <meta itemprop="url" content="http://mtw.github.io" />
    <article class="hentry" role="article" itemprop="blogPost" itemscope itemtype="http://schema.org/BlogPosting">
      
  <header class="page-header">
    
      <p class="meta text-muted text-uppercase">
        












<span class="glyphicon glyphicon-calendar"></span> <time datetime="2014-04-13T22:22:41+02:00"  data-updated="true" itemprop="datePublished dateCreated">2014-04-13</time>
        
      </p>
    
    
    <h1 class="entry-title" itemprop="name headline">
        TSSAR: TSS Annotation Regime for dRNA-seq Data
        
    </h1>
    
  </header>


<div class="entry-content clearfix" itemprop="articleBody description"><p>In silico identification of bacterial transcription start sites (TSS) has
been a major challenge for the last years. To address this issue, we have
developped <a href="http://rna.tbi.univie.ac.at/TSSAR">TSSAR</a>, a statistical method
for analyzing dRNA-seq data, together with colleagues from the
Bioinformatics department at the University of Leipzig.</p>

<p>The TSSAR method paper is now out: <a href="http://dx.doi.org/10.1186/1471-2105-15-89">TSSAR: TSS annotation regime for dRNA-seq
data</a> <em>BMC Bioinformatics</em> 2014
<strong>15</strong>:89</p>

<blockquote><h4>Abstract</h4>

<p>Background</p>

<p>Differential RNA sequencing dRNA-seq is a high-throughput screening
technique designed to examine the architecture of bacterial operons in
general and the precise position of transcription start sites (TSS) in
particular. Hitherto, dRNA-seq data were analyzed by visualizing the
sequencing reads mapped to the reference genome and manually annotating
reliable positions. This is very labor intensive and, due to the
subjectivity, biased.</p>

<p>Results</p>

<p>Here, we present <strong>TSSAR</strong>, a tool for automated <em>de-novo</em> TSS annotation
from dRNA-seq data that respects the statistics of dRNA-seq
libraries. <strong>TSSAR</strong> uses the premise that the number of sequencing reads
starting at a certain genomic position within a transcriptional active
region follows a Poisson distribution with a parameter that depends on the
local strength of expression. The differences of two dRNA-seq library
counts thus follow a Skellam distribution. This provides a statistical
basis to identify significantly enriched primary transcripts.</p>

<p>We assessed the performance by analyzing a publicly available dRNA-seq
  data set using <strong>TSSAR</strong> and two simple approaches that utilize
  user-defined score cutoffs. We evaluated the power of reproducing the
  manual TSS annotation. Furthermore, the same data set was used to
  reproduce 74 experimentally validated TSS in <em>H. pylori</em> from reliable
  techniques such as RACE or primer extension. Both analyses showed that
  <strong>TSSAR</strong> outperforms the static cutoff-dependent approaches.</p>

<p>Conclusions</p>

<p>Having an automated and efficient tool for analyzing dRNA-seq data
  facilitates the use of the dRNA-seq technique and promotes its
  application to more sophisticated analysis. For instance, monitoring the
  plasticity and dynamics of the transcriptomal architecture triggered by
  different stimuli and growth conditions becomes possible.</p>

<p>The main asset of a novel tool for dRNA-seq analysis that reaches out to
a broad user community is usability. As such, we provide <strong>TSSAR</strong> both as
intuitive RESTful Web service [<a href="http://rna.tbi.univie.ac.at/TSSAR">http://rna.tbi.univie.ac.at/TSSAR</a>] together
with a set of post-processing and analysis tools, as well as a
stand-alone version for use in high-throughput dRNA-seq data analysis
pipelines.</p>

<p>Keywords</p>

<p>Differential RNA sequencing, dRNA-seq, TSS, Transcription start site
annotation, Transcriptome, RESTful Web service, Next generation sequencing</p></blockquote>
</div>


      <footer>
        <p class="meta text-muted">
          
  

<span class="glyphicon glyphicon-user"></span> <span class="byline author vcard" itemprop="author" itemscope itemtype="http://schema.org/Person">Posted by <span class="fn" itemprop="name">Michael T. Wolfinger</span></span>

          












<span class="glyphicon glyphicon-calendar"></span> <time datetime="2014-04-13T22:22:41+02:00"  data-updated="true" itemprop="datePublished dateCreated">2014-04-13</time>
          

<span class="glyphicon glyphicon-tags"></span>&nbsp;
<span class="categories">
  
    <a class='category' href='/blog/categories/publications/'>publications</a>
  
</span>


        </p>
        
          <div class="sharing">
  
  
  
    <div class="fb-like" data-send="true" data-width="450" data-show-faces="false"></div>
  
</div>

        
        
          <ul class="meta text-muted pager">
            
            <li class="previous"><a href="/blog/2013/12/18/tivoli-storage-manager-under-debian-7-wheezy/" title="Previous Post: Tivoli Storage Manager under Debian 7 wheezy">&laquo; Tivoli Storage Manager under Debian 7 wheezy</a></li>
            
            
            <li class="next"><a href="/blog/2014/04/15/how-to-compute-normalized-rna-seq-expression-from-multicov-files/" title="Next Post: How to compute normalized RNA-seq expression from multicov files">How to compute normalized RNA-seq expression from multicov files &raquo;</a></li>
            
          </ul>
        
      </footer>
    </article>
    
  </div>

  
  <aside class="sidebar col-md-3">
    
      <section class="panel panel-default">
  <div class="panel-heading">
    <h3 class="panel-title">Recent Posts</h3>
  </div>
  
  <div id="recent_posts" class="list-group">
    
    <a class="list-group-item " href="/blog/2015/03/03/viennangs-installation-howto/">ViennaNGS Installation HOWTO</a>
    
    <a class="list-group-item " href="/blog/2015/03/02/viennangs-a-toolbox-for-building-efficient-next-generation-sequencing-analysis-pipelines/">ViennaNGS: A Toolbox for Building Efficient Next-generation Sequencing Analysis Pipelines</a>
    
    <a class="list-group-item " href="/blog/2014/04/15/how-to-compute-normalized-rna-seq-expression-from-multicov-files/">How to Compute Normalized RNA-seq Expression From Multicov Files</a>
    
    <a class="list-group-item active" href="/blog/2014/04/13/tssar-tss-annotation-regime-for-drna-seq-data/">TSSAR: TSS Annotation Regime for dRNA-seq Data</a>
    
    <a class="list-group-item " href="/blog/2013/12/18/tivoli-storage-manager-under-debian-7-wheezy/">Tivoli Storage Manager Under Debian 7 Wheezy</a>
    
  </div>
</section>

<section class="twitterOct">
  <h1 style="margin-bottom: 0.4em"> Tweets </h1>
  <a class="twitter-timeline"
	 data-dnt="true" href="https://twitter.com/mtwolfinger" 
     data-widget-id="412878550417420288">
     
     Tweets by @mtwolfinger
  </a>
  <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
</section>


    
  </aside>
  
</div>

        </div>
      </div>
    </div>
    <footer role="contentinfo"><div class="container">
    <p class="text-muted credits">
  Copyright &copy; 2015 - Michael T. Wolfinger<br>
  <small>
      <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>,
      <span class="credit">customized with <a href="https://github.com/kAworu/octostrap3">octostrap3</a></span>.
  </small>
</p>

</div>
</footer>
    



<div id="fb-root"></div>
<script>(function(d, s, id) {
  var js, fjs = d.getElementsByTagName(s)[0];
  if (d.getElementById(id)) {return;}
  js = d.createElement(s); js.id = id; js.async = true;
  js.src = "//connect.facebook.net/en_US/all.js#appId=212934732101925&xfbml=1";
  fjs.parentNode.insertBefore(js, fjs);
}(document, 'script', 'facebook-jssdk'));</script>






<script src="/assets/bootstrap/dist/js/bootstrap.min.js"></script>
<script src="/javascripts/modernizr.js"></script>


  </body>
</html>
