<!DOCTYPE html>
<html lang="en" prefix="og: http://ogp.me/ns#">
<head>
  <meta charset="UTF-8" />
  <title>How to compute normalized RNA-seq expression from multicov files | michaelwolfinger.com</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300;400|Source+Code+Pro:200:300,400|Source+Sans+Pro:300,400" />
  <link rel="stylesheet" href="https://michaelwolfinger.com/static/m-light.compiled.css" />
  <link rel="stylesheet" href="https://michaelwolfinger.com/static/academicons.min.css" />
  <link rel="stylesheet" href="https://michaelwolfinger.com/static/m-mtw.css" />
  <link rel="canonical" href="https://michaelwolfinger.com/blog/2014/How-to-compute-normalized-RNA-seq-expression-from-multicov-files/" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <meta name="theme-color" content="#EAEAEA" />
  <meta name="twitter:site" content="@mtwolfinger" />
  <meta name="twitter:site:id" content="15105886" />
  <meta property="og:site_name" content="michaelwolfinger.com" />
  <meta property="og:title" content="How to compute normalized RNA-seq expression from multicov files" />
  <meta name="twitter:title" content="How to compute normalized RNA-seq expression from multicov files" />
  <meta property="og:url" content="https://michaelwolfinger.com/blog/2014/How-to-compute-normalized-RNA-seq-expression-from-multicov-files/" />
  <meta property="og:description" content="Why you should use TPM rather than RPKM" />
  <meta name="twitter:description" content="Why you should use TPM rather than RPKM" />
  <meta name="twitter:card" content="summary" />
  <meta property="og:type" content="article" />
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-T2RW3VW55N"></script>
  <script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-T2RW3VW55N');
  </script>
</head>
<body>
<header><nav id="navigation">
  <div class="m-container">
    <div class="m-row">
      <a href="../../../" id="m-navbar-brand" class="m-col-t-9 m-col-m-none m-left-m">Michael T. Wolfinger</a>
      <a id="m-navbar-show" href="#navigation" title="Show navigation" class="m-col-t-3 m-hide-m m-text-right"></a>
      <a id="m-navbar-hide" href="#" title="Hide navigation" class="m-col-t-3 m-hide-m m-text-right"></a>
      <div id="m-navbar-collapse" class="m-col-t-12 m-show-m m-col-m-none m-right-m">
        <div class="m-row">
          <ol class="m-col-t-12 m-col-m-none">
            <li>
              <a href="https://michaelwolfinger.com/publications">Publications</a>
              <ol>
                <li><a href="https://michaelwolfinger.com/publications">Papers</a></li>
                <li><a href="https://michaelwolfinger.com/publications/presentations">Presentations</a></li>
                <li><a href="https://michaelwolfinger.com/publications/posters">Posters</a></li>
              </ol>
            </li>
            <li><a href="https://michaelwolfinger.com/teaching">Teaching</a></li>
            <li><a href="https://michaelwolfinger.com/consulting">Consulting</a></li>
            <li><a href="https://michaelwolfinger.com/blog/" id="m-navbar-current">Blog</a></li>
            <li><a href="https://michaelwolfinger.com/contact">Contact</a></li>
          </ol>
        </div>
      </div>
    </div>
  </div>
</nav></header>
<main>
<div class="m-container">
  <div class="m-row">
    <article class="m-col-m-10 m-nopadb">
      <header>
        <h1><a href="https://michaelwolfinger.com/blog/2014/How-to-compute-normalized-RNA-seq-expression-from-multicov-files/" rel="bookmark" title="Permalink to How to compute normalized RNA-seq expression from multicov files">
          <time class="m-date" datetime="2014-04-15T00:00:00+02:00">
            Apr <span class="m-date-day">15</span> 2014
          </time>
          How to compute normalized RNA-seq expression from multicov files
        </a></h1>
        <p>Why you should use TPM rather than RPKM</p>
      </header>
      <div class="m-clearfix-l"></div>
<!-- content -->
<aside class="m-block m-default">
<h3>Abstract</h3>
<p>Whenever it comes to analyzing RNA-seq experiments, there is a need for
comparing expression data at a quantitative level. Consider a scenario
where samples were taken from different conditions and subjected to
Illumina sequencing. Whether those samples were multiplexed or sequenced on
a single lane each, one generally gets a different number of raw reads from
each sample, refelcting experimental and technical biases inherent in the
RNA-seq protocols. Various measures for normalization of RNA-seq samples
have been proposed, the most widely used being RPKM (reads per kilobase per
million). While RPKM tries to account for different sequencing depth by
normalizing by the number of reads sequenced in a specific sample, divided
by 10^6, this very step causes a systematic bias, as has been shown here:</p>
</aside>
<aside class="m-block m-info">
<h3>Reference</h3>
<p>
<a class="m-flat m-text m-strong" href="http://doi.org/10.1007/s12064-012-0162-3">Measurement of mRNA abundance using RNA-seq data: RPKM measure is inconsistent among samples</a><br />
GÃ¼nter P. Wagner, Koryu Kin, Vincent J. Lynch<br />
<em>Theory Biosci.</em> 131, 281-285 (2012) | <a class="doi" href="https://doi.org/10.1007/s12064-012-0162-3">doi:10.1007/s12064-012-0162-3</a><br />
</p>
</aside>
<aside class="m-block m-info">
<h3>Reference</h3>
<p>
<a class="m-flat m-text m-strong" href="http://doi.org/10.1093/bioinformatics/btp692">RNA-Seq gene expression estimation with read mapping uncertainty</a><br />
Bo Li, Victor Ruotti, Ron M. Stewart, James A. Thomson, Colin N. Dewey<br />
<em>Bioinformatics</em> 6(4), 493-500. (2009) | <a class="doi" href="https://doi.org/10.1093/bioinformatics/btp692">doi:10.1093/bioinformatics/btp692</a><br />
</p>
</aside>
<p>The central point of these papers is to work out an alternative measure for
RNA-seq expression abundance that resembles as closely as possible the
<em>relative molar concentraction</em> (rmc) of each RNA species present in a
sample. It is easy to see that the average rmc across genes has to be a
constant that only depends on the number of genes mapped in an RNA-seq
experiment.</p>
<p>One example of measures that fulfills the invariant average criterion is
<em>Transcript per million</em> (TPM), being defined as</p>
<div class="m-math">
<svg version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' width='78.922067pt' height='15.969084pt' viewBox='-.956413 -11.827084 78.922067 15.969084'>
<defs>
<path id='g21-48' d='M4.157561-2.882343C4.157561-4.672889 3.362733-5.904436 2.218531-5.904436C1.73814-5.904436 1.371296-5.755951 1.048125-5.450248C.541531-4.961124 .209625-3.956671 .209625-2.934749C.209625-1.982703 .497859-.960781 .908375-.471656C1.231547-.087344 1.677 .122281 2.183593 .122281C2.629046 .122281 3.004624-.026203 3.319062-.331906C3.825655-.812297 4.157561-1.825484 4.157561-2.882343ZM3.319062-2.864874C3.319062-1.03939 2.934749-.104812 2.183593-.104812S1.048125-1.03939 1.048125-2.85614C1.048125-4.707827 1.441171-5.677342 2.192327-5.677342C2.926015-5.677342 3.319062-4.690358 3.319062-2.864874Z'/>
<path id='g21-49' d='M3.441343 0V-.131016C2.751327-.13975 2.611577-.227094 2.611577-.646344V-5.886967L2.541702-5.904436L.969515-5.109608V-4.987327C1.074328-5.030999 1.170406-5.065936 1.205343-5.083405C1.362562-5.144545 1.511046-5.179483 1.59839-5.179483C1.781812-5.179483 1.860421-5.048467 1.860421-4.768967V-.812297C1.860421-.524062 1.790546-.323172 1.650796-.244562C1.519781-.165953 1.3975-.13975 1.030656-.131016V0H3.441343Z'/>
<path id='g21-54' d='M4.087686-1.912828C4.087686-3.030827 3.450077-3.738311 2.445624-3.738311C2.061312-3.738311 1.87789-3.677171 1.327625-3.345265C1.563453-4.664155 2.541702-5.607467 3.912999-5.834561L3.89553-5.974311C2.899812-5.886967 2.393218-5.721014 1.755609-5.275561C.812297-4.603014 .296969-3.607296 .296969-2.43689C.296969-1.677 .532797-.908375 .908375-.471656C1.240281-.087344 1.711937 .122281 2.253468 .122281C3.33653 .122281 4.087686-.707484 4.087686-1.912828ZM3.301593-1.615859C3.301593-.655078 2.960952-.122281 2.349546-.122281C1.580921-.122281 1.109265-.943312 1.109265-2.29714C1.109265-2.742593 1.17914-2.987155 1.353828-3.118171C1.53725-3.257921 1.808015-3.33653 2.113718-3.33653C2.864874-3.33653 3.301593-2.707655 3.301593-1.615859Z'/>
<path id='g3-61' d='M6.574598-3.930466V-4.597454H.47642V-3.930466H6.574598ZM6.574598-1.512634V-2.179622H.47642V-1.512634H6.574598Z'/>
<path id='g5-3' d='M4.871396-3.978108C4.871396-4.180587 4.704649-4.394976 4.49026-4.394976C4.216318-4.394976 3.239657-3.418315 2.894252-3.215836C2.894252-3.882824 3.227747-4.549812 3.227747-4.95477C3.227747-5.20489 3.060999-5.359727 2.810879-5.359727C2.572669-5.359727 2.429743-5.19298 2.429743-4.95477C2.429743-4.49026 2.715595-3.859003 2.715595-3.215836C2.334459-3.442136 1.405439-4.442618 1.155319-4.442618C.94093-4.442618 .750362-4.240139 .750362-4.02575C.750362-3.608883 2.155801-3.323031 2.620311-3.060999C2.12007-2.775147 1.19105-2.632221 .95284-2.477385C.833735-2.394011 .738451-2.286817 .738451-2.13198C.738451-1.917591 .905198-1.727023 1.119587-1.727023C1.393529-1.727023 2.35828-2.691774 2.715595-2.894252C2.715595-2.239175 2.382101-1.572186 2.382101-1.155319C2.382101-.917109 2.560758-.750362 2.798968-.750362S3.180104-.929019 3.180104-1.155319C3.180104-1.64365 2.906163-2.262996 2.906163-2.894252C3.275389-2.679863 4.216318-1.679381 4.454528-1.679381C4.668917-1.679381 4.859486-1.88186 4.859486-2.096249C4.859486-2.251085 4.752291-2.35828 4.633186-2.429743C4.323513-2.632221 3.501688-2.763237 2.989536-3.060999C3.54933-3.394494 4.311602-3.442136 4.657007-3.644614C4.788022-3.716077 4.871396-3.823272 4.871396-3.978108Z'/>
<path id='g8-54' d='M2.286223-.41925C1.997989 .694383 1.663899 1.303606 1.093981 1.303606C.687832 1.303606 .831949 .956414 .50441 .956414C.432352 .956414 .255481 1.054676 .255481 1.218446C.255481 1.493579 .779543 1.545985 1.028473 1.545985C1.270852 1.545985 1.696653 1.473926 2.00454 1.152938C2.384485 .75334 2.59411 .49786 2.856141-.661629L3.314696-2.823388C3.157477-2.823388 2.980606-2.803735 2.849591-2.836489C2.67272-2.856141 2.587559-2.875794 2.318977-2.875794C1.604942-2.875794 .550266-1.92593 .550266-.845051C.550266-.478207 .694383 .072059 1.244649 .072059C1.59184 .072059 1.991438-.294785 2.384485-.877805L2.286223-.41925ZM2.502399-1.434622C2.187962-.845051 1.663899-.294785 1.454274-.294785C1.231547-.294785 1.113633-.58957 1.113633-.956414C1.113633-1.650797 1.742508-2.666169 2.33863-2.666169C2.476196-2.666169 2.613762-2.626864 2.738227-2.548255C2.738227-2.548255 2.607212-1.84077 2.502399-1.434622Z'/>
<use id='g24-48' xlink:href='#g21-48' transform='scale(1.363636)'/>
<use id='g24-49' xlink:href='#g21-49' transform='scale(1.363636)'/>
<path id='g12-34' d='M5.721014-.75989C5.694811-.532797 5.581264-.270766 5.345436-.253297L5.118342-.235828C5.083405-.235828 5.030999-.209625 5.030999-.165953L5.01353 0L5.030999 .017469C5.43278 .008734 5.686076 0 5.930639 0C6.183936 0 6.480904 .008734 6.873951 .017469L6.89142 0L6.908889-.165953C6.917623-.209625 6.882685-.227094 6.830279-.235828L6.603186-.253297C6.393561-.270766 6.341154-.384312 6.341154-.567734C6.341154-.628875 6.349889-.690015 6.358623-.75989L6.89142-5.01353C6.917623-5.240623 7.03117-5.362905 7.284467-5.380373L7.51156-5.397842C7.555232-5.397842 7.607638-5.397842 7.616373-5.458983L7.633842-5.63367L7.616373-5.651139H6.515842L3.869327-1.205343C3.816921-1.118 3.790718-1.083062 3.773249-1.083062C3.747046-1.083062 3.738311-1.118 3.729577-1.196609L2.873609-5.642405L1.615859-5.651139L1.59839-5.63367L1.580921-5.458983C1.580921-5.424045 1.589656-5.397842 1.650796-5.397842L1.886624-5.380373C2.087515-5.362905 2.166124-5.310498 2.166124-5.188217C2.166124-5.135811 2.15739-5.083405 2.139921-5.01353L.908375-.751156C.847234-.515328 .733687-.270766 .471656-.253297L.244562-.235828C.192156-.235828 .157219-.209625 .157219-.165953L.13975 0L.157219 .017469C.410516 .008734 .69875 0 .89964 0S1.423703 .008734 1.668265 .017469L1.694468 0L1.703203-.165953C1.703203-.209625 1.677-.235828 1.633328-.235828L1.423703-.253297C1.25775-.262031 1.196609-.375578 1.196609-.524062C1.196609-.593937 1.205343-.672547 1.231547-.751156L2.332077-4.576811H2.349546L3.19678-.043672C3.214249 .017469 3.257921 .052406 3.310327 .052406C3.353999 .052406 3.397671 .026203 3.441343-.043672L6.227608-4.664155H6.245076L5.721014-.75989Z'/>
<path id='g12-37' d='M2.541702-5.01353C2.585374-5.266827 2.698921-5.415311 3.301593-5.415311C3.799452-5.415311 4.271108-5.258092 4.271108-4.541874C4.271108-4.410858 4.253639-4.262374 4.218702-4.087686C4.03528-3.144374 3.511218-2.760062 2.698921-2.760062C2.489296-2.760062 2.201062-2.77753 2.113718-2.803734L2.541702-5.01353ZM1.004453-.62014C.960781-.375578 .812297-.2795 .471656-.253297L.253297-.235828C.209625-.235828 .183422-.209625 .174687-.165953L.13975 0L.157219 .017469C.462922 .008734 .882172 0 1.231547 0C1.563453 0 1.982703 .008734 2.375749 .017469L2.393218 0L2.428156-.165953C2.43689-.209625 2.410687-.235828 2.375749-.235828L2.070046-.253297C1.799281-.270766 1.677-.331906 1.677-.506594C1.677-.541531 1.677-.576469 1.685734-.62014L2.061312-2.550437C2.235999-2.489296 2.384484-2.463093 2.681452-2.463093C4.209968-2.463093 4.847577-3.502483 4.987327-4.244905C5.004795-4.332249 5.01353-4.419592 5.01353-4.524405C5.01353-5.039733 4.716561-5.71228 3.423874-5.71228C3.205515-5.71228 2.663984-5.63367 2.323343-5.63367C1.991437-5.63367 1.563453-5.642405 1.25775-5.651139L1.231547-5.63367L1.196609-5.450248C1.187875-5.415311 1.205343-5.397842 1.240281-5.397842L1.563453-5.380373C1.685734-5.371639 1.869156-5.310498 1.869156-5.100873C1.869156-5.07467 1.869156-5.039733 1.860421-5.01353L1.004453-.62014Z'/>
<path id='g12-41' d='M2.000171-5.63367C1.773078-5.63367 1.589656-5.71228 1.380031-5.747217C1.266484-5.319233 1.161672-5.065936 .969515-4.611749C1.03939-4.568077 1.152937-4.541874 1.231547-4.541874C1.484843-5.118342 1.703203-5.29303 2.166124-5.29303H2.768796C3.109437-5.29303 3.161843-5.205686 3.118171-4.952389L2.288406-.663812C2.235999-.41925 2.104984-.296969 1.746874-.262031L1.476109-.235828C1.441171-.200891 1.423703-.061141 1.45864 .017469C1.851687 .008734 2.166124 0 2.506765 0C2.838671 0 3.231718 .008734 3.61603 .017469C3.642233-.017469 3.668436-.192156 3.642233-.235828L3.345265-.262031C3.004624-.288234 2.908546-.401781 2.960952-.663812L3.790718-4.926186C3.851858-5.223155 3.965405-5.29303 4.262374-5.29303H4.73403C5.188217-5.29303 5.319233-5.065936 5.467717-4.480733C5.555061-4.480733 5.624936-4.480733 5.729748-4.524405C5.63367-5.022264 5.607467-5.397842 5.63367-5.747217C5.345436-5.694811 5.231889-5.63367 4.917452-5.63367H2.000171Z'/>
<path id='g12-54' d='M3.048296-.559C2.663984 .925843 2.218531 1.73814 1.45864 1.73814C.917109 1.73814 1.109265 1.275218 .672547 1.275218C.576469 1.275218 .340641 1.3975 .340641 1.615859C.340641 1.982703 1.030656 2.052578 1.353828 2.052578C1.668265 2.052578 2.235999 1.956499 2.63778 1.528515C3.13564 .995718 3.41514 .663812 3.75578-.882172L4.393389-3.764515C4.183764-3.764515 3.956671-3.738311 3.781983-3.781983C3.546155-3.808186 3.432608-3.834389 3.083233-3.834389C2.148656-3.834389 .75989-2.57664 .75989-1.135468C.75989-.646344 .952047 .087344 1.685734 .087344C2.148656 .087344 2.681452-.393047 3.205515-1.170406L3.048296-.559ZM3.33653-1.912828C2.908546-1.126734 2.201062-.393047 1.921562-.393047C1.624593-.393047 1.45864-.786094 1.45864-1.275218C1.45864-2.201062 2.305874-3.563624 3.109437-3.563624C3.292858-3.563624 3.485015-3.502483 3.650968-3.397671C3.650968-3.397671 3.47628-2.454359 3.33653-1.912828Z'/>
<path id='g12-67' d='M1.624593-3.747046L1.161672-3.720843C.995718-3.712108 .934578-3.642233 .917109-3.572358C.908375-3.546155 .882172-3.441343 .882172-3.423874S.908375-3.406405 .934578-3.406405H1.545984L1.013187-1.030656C.952047-.751156 .890906-.48039 .890906-.314437C.890906 0 1.048125 .087344 1.31889 .087344C1.711937 .087344 2.288406-.270766 2.585374-.812297C2.559171-.925843 2.498031-.97825 2.393218-.97825C1.991437-.524062 1.834218-.436719 1.694468-.436719C1.668265-.436719 1.580921-.454187 1.580921-.559C1.580921-.637609 1.607125-.812297 1.677-1.109265L2.192327-3.406405H3.048296C3.153108-3.450077 3.231718-3.677171 3.13564-3.747046H2.270937L2.358281-4.140093C2.393218-4.306046 2.471827-4.559342 2.471827-4.664155C2.471827-4.812639 2.401952-4.899983 2.122453-4.899983C1.808015-4.899983 1.825484-4.506936 1.73814-4.183764L1.624593-3.747046Z'/>
<use id='g16-34' xlink:href='#g12-34' transform='scale(1.363636)'/>
<use id='g16-37' xlink:href='#g12-37' transform='scale(1.363636)'/>
<use id='g16-41' xlink:href='#g12-41' transform='scale(1.363636)'/>
</defs>
<g id='page1'>
<use x='-.956413' y='0' xlink:href='#g16-41'/>
<use x='7.041587' y='0' xlink:href='#g16-37'/>
<use x='14.017415' y='0' xlink:href='#g16-34'/>
<use x='23.250789' y='1.793263' xlink:href='#g12-54'/>
<use x='31.619871' y='0' xlink:href='#g3-61'/>
<use x='42.906825' y='-6.927103' xlink:href='#g12-67'/>
<use x='45.576414' y='-5.612042' xlink:href='#g8-54'/>
<rect x='43.213674' y='-3.407221' height='.669486' width='6.289905'/>
<use x='43.0096' y='4.124532' xlink:href='#g12-41'/>
<use x='53.355755' y='0' xlink:href='#g5-3'/>
<use x='61.6433' y='0' xlink:href='#g24-49'/>
<use x='67.620884' y='0' xlink:href='#g24-48'/>
<use x='73.598468' y='-4.339714' xlink:href='#g21-54'/>
</g>
</svg>
</div><p>where t_g is a proxy for the number of transcripts that can be explained by
a certain number of mapped reads and T is the sum of all t_g over all
genes. If one is interested in mRNA abundance, the  average TPM - and thus
the average rmc is inversely proportional to the number of features
present in a reference annotation.</p>
<p>Practically, TPM values for individual genes can be computed from read
count tables, ie. tables that give the number of reads overlapping a
specific gene. Typical programs for obtaining read count tables are
<a href="http://htseq.readthedocs.io/">htseq-count</a>
or
<a href="http://bedtools.readthedocs.org/en/latest/content/tools/multicov.html">multiBamCov</a>
(see <a href="http://bedtools.readthedocs.org/en/latest/index.html">bedtools</a> multicov).</p>
<p>I have recently implemented
<a href="https://github.com/mtw/ViennaNGS/blob/master/scripts/normalize_multicov.pl">normalize_multicov.pl</a>,
a tool for computing normalized RNA-seq expression in terms of TPM from
multicov files. It is part of the
<a href="https://github.com/mtw/ViennaNGS">ViennaNGS</a> Perl Modules for NGS analysis
and very easy to use: Just provide it the output of a bedtols multicov run
on your data as well as the read length used for sequencing your samples
and get back a normalized multicov file of your samples in terms of
TPM. That's all ...</p>
<!-- /content -->
      <footer>
        <p>Posted by Michael T. Wolfinger on <time datetime="2014-04-15T00:00:00+02:00">Tue 15 April 2014</time>. <span class="m-label m-success">updated <time datetime="2022-02-09T00:00:00+01:00">Wed 09 February 2022</time></span>
      </footer>
    </article>
    <nav class="m-navpanel m-col-m-2">
      <h3>Categories</h3>
      <ol class="m-block-bar-m">
        <li><a href="https://michaelwolfinger.com/blog/category/howto/">HOWTO</a></li>
        <li><a href="https://michaelwolfinger.com/blog/category/publications/">publications</a></li>
      </ol>
      <h3>Tag cloud</h3>
      <ul class="m-tagcloud">
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/3d/">3D</a></li>
        <li class="m-tag-1"><a href="https://michaelwolfinger.com/blog/tag/bacteria/">bacteria</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/energy-landscapes/">energy landscapes</a></li>
        <li class="m-tag-5"><a href="https://michaelwolfinger.com/blog/tag/ngs/">NGS</a></li>
        <li class="m-tag-5"><a href="https://michaelwolfinger.com/blog/tag/non-coding-rna/">non-coding RNA</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/rna-kinetics/">RNA kinetics</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/shape/">SHAPE</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/viennarna/">ViennaRNA</a></li>
        <li class="m-tag-4"><a href="https://michaelwolfinger.com/blog/tag/viruses/">viruses</a></li>
      </ul>
    </nav>
  </div>
</div>
</main>
<footer><nav>
  <div class="m-container">
    <div class="m-row">
      <div class="m-col-s-3 m-col-t-6">
        <h3><a href="https://michaelwolfinger.com/">Home</a></h3>
        <ul>
          <li><a href="https://michaelwolfinger.com/teaching">Teaching</a></li>
          <li><a href="https://michaelwolfinger.com/consulting">Consulting</a></li>
          <li><a href="https://michaelwolfinger.com/blog">Blog</a></li>
          <li><a href="https://michaelwolfinger.com/contact">Contact</a></li>
        </ul>
      </div>
      <div class="m-col-s-3 m-col-t-6">
        <h3><a href="https://michaelwolfinger.com/publications">Publications</a></h3>
        <ul>
          <li><a href="https://michaelwolfinger.com/publications/presentations">Presentations</a></li>
          <li><a href="https://michaelwolfinger.com/publications/posters">Posters</a></li>
        </ul>
      </div>
      <div class="m-clearfix-t"></div>
      <div class="m-col-s-3 m-col-t-6">
        <h3>My Profiles</h3>
        <ul>
          <li><a href="https://scholar.google.at/citations?user=w0PHGnEAAAAJ&amp;hl=en">Google Scholar</a></li>
          <li><a href="https://publons.com/researcher/2466680/michael-thomas-wolfinger/">Publons</a></li>
          <li><a href="https://github.com/mtw">GitHub</a></li>
          <li><a href="https://orcid.org/0000-0003-0925-5205">ORCID</a></li>
        </ul>
      </div>
      <div class="m-col-s-3 m-col-t-6">
        <h3>Social</h3>
        <ul>
          <li><a href="https://www.researchgate.net/profile/Michael-Wolfinger">ResearchGate</a></li>
          <li><a href="https://twitter.com/mtwolfinger">Twitter</a></li>
          <li><a href="https://www.linkedin.com/in/michaelwolfinger/">Linkedin</a></li>
        </ul>
      </div>
    </div>
    <div class="m-row">
      <div class="m-col-l-10 m-push-l-1">
        <p>
        michaelwolfinger.com - Copyright Â© Michael T. Wolfinger, 2021-2022<br />
        </p>
      </div>
    </div>
  </div>
</nav></footer>
</body>
</html>