<!DOCTYPE html>
<html lang="en" prefix="og: http://ogp.me/ns#">
<head>
  <meta charset="UTF-8" />
  <title>RNA-protein complex refinement using AI modeling and docking</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300;400|Source+Code+Pro:200:300,400|Source+Sans+Pro:300,400" />
  <link rel="stylesheet" href="https://michaelwolfinger.com/static/m-light.compiled.min.css" />
  <link rel="stylesheet" href="https://michaelwolfinger.com/static/m-mtw.min.css" />
  <link rel="canonical" href="https://michaelwolfinger.com/blog/2023/blog/2023-06-09-A-Structural-Refinement-Technique-for-Protein-RNA-Complexes-Using-Combination-of-AI-based-Modeling-and-Flexible-Docking-A-Study-of-Musashi-1-Protein/" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <meta name="theme-color" content="#EAEAEA" />
  <meta name="description" content="Enhance protein-RNA complex modeling with AI-based techniques and PaCS-MD for faster conformational sampling and improved quality" />
  <meta name="twitter:site" content="@mtwolfinger" />
  <meta name="twitter:site:id" content="15105886" />
  <meta property="og:site_name" content="michaelwolfinger.com Bioinformatics Blog" />
  <meta property="og:title" content="RNA-protein complex refinement using AI modeling and docking" />
  <meta name="twitter:title" content="RNA-protein complex refinement using AI modeling and docking" />
  <meta property="og:url" content="https://michaelwolfinger.com/blog/2023/blog/2023-06-09-A-Structural-Refinement-Technique-for-Protein-RNA-Complexes-Using-Combination-of-AI-based-Modeling-and-Flexible-Docking-A-Study-of-Musashi-1-Protein/" />
  <meta property="og:description" content="This study presents an efficient technique for refining protein-RNA complexes using AI-based modeling and flexible docking. The method, utilizing parallel cascade selection molecular dynamics (PaCS-MD), accelerates conformational sampling of flexible RNA regions and produces high-quality complex models. Experimental validation demonstrates its superiority over template-based modeling, suggesting its potential for constructing complexes with non-canonical RNA-protein interactions" />
  <meta name="twitter:description" content="This study presents an efficient technique for refining protein-RNA complexes using AI-based modeling and flexible docking. The method, utilizing parallel cascade selection molecular dynamics (PaCS-MD), accelerates conformational sampling of flexible RNA regions and produces high-quality complex models. Experimental validation demonstrates its superiority over template-based modeling, suggesting its potential for constructing complexes with non-canonical RNA-protein interactions" />
  <meta name="twitter:card" content="summary" />
  <meta property="og:type" content="article" />
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-T2RW3VW55N"></script>
  <script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-T2RW3VW55N');
  </script>
</head>
<body>
<header><nav id="navigation">
  <div class="m-container">
    <div class="m-row">
      <a href="../../../../" id="m-navbar-brand" class="m-col-t-9 m-col-m-none m-left-m">Michael T. Wolfinger</a>
      <a id="m-navbar-show" href="#navigation" title="Show navigation" class="m-col-t-3 m-hide-m m-text-right"></a>
      <a id="m-navbar-hide" href="#" title="Hide navigation" class="m-col-t-3 m-hide-m m-text-right"></a>
      <div id="m-navbar-collapse" class="m-col-t-12 m-show-m m-col-m-none m-right-m">
        <div class="m-row">
          <ol class="m-col-t-12 m-col-m-none">
            <li>
              <a href="https://michaelwolfinger.com/research">Research</a>
              <ol>
                <li><a href="https://michaelwolfinger.com/team">Team</a></li>
              </ol>
            </li>
            <li>
              <a href="https://michaelwolfinger.com/publications">Publications</a>
              <ol>
                <li><a href="https://michaelwolfinger.com/publications">Papers</a></li>
                <li><a href="https://michaelwolfinger.com/publications/presentations">Presentations</a></li>
                <li><a href="https://michaelwolfinger.com/publications/posters">Posters</a></li>
              </ol>
            </li>
            <li><a href="https://michaelwolfinger.com/teaching">Teaching</a></li>
            <li><a href="https://michaelwolfinger.com/consulting">Consultancy</a></li>
            <li><a href="https://michaelwolfinger.com/contact">Contact</a></li>
            <li><a href="https://michaelwolfinger.com/blog/" id="m-navbar-current">Blog</a></li>
          </ol>
        </div>
      </div>
    </div>
  </div>
</nav></header>
<main>
<div class="m-container">
  <div class="m-row">
    <article class="m-col-m-10 m-nopadb">
      <header>
        <h1><a href="https://michaelwolfinger.com/blog/2023/blog/2023-06-09-A-Structural-Refinement-Technique-for-Protein-RNA-Complexes-Using-Combination-of-AI-based-Modeling-and-Flexible-Docking-A-Study-of-Musashi-1-Protein/" rel="bookmark" title="Permalink to RNA-protein complex refinement using AI modeling and docking">
          <time class="m-date" datetime="2023-06-09T00:00:00+02:00">
            Jun <span class="m-date-day">09</span> 2023
          </time>
          RNA-protein complex refinement using AI modeling and docking
        </a></h1>
        <p>This study presents an efficient technique for refining protein-RNA complexes using AI-based modeling and flexible docking. The method, utilizing parallel cascade selection molecular dynamics (PaCS-MD), accelerates conformational sampling of flexible RNA regions and produces high-quality complex models. Experimental validation demonstrates its superiority over template-based modeling, suggesting its potential for constructing complexes with non-canonical RNA-protein interactions</p>
      </header>
      <div class="m-clearfix-l"></div>
<!-- content -->
<p>Understanding the intimate dance between proteins and RNA is essential for unraveling the mysteries of cellular functions. In a recent study, we proposed an innovative and efficient technique to accurately model the interactions between proteins and RNA, leading to insights that have the potential to advance the fields of structural biology and drug design.</p>
<section id="the-challenge-building-complex-structures">
<h2>The Challenge: Building Complex Structures</h2>
<p>When a protein interacts with RNA, it forms what is known as a protein-RNA complex. Constructing accurate models of these complexes is a formidable challenge. Traditional methods, such as template-based modeling, involve creating the overall conformation and then manually connecting RNA with a protein’s RNA-binding domains (RBDs). However, this doesn't always lead to natural or biologically plausible complexes.</p>
</section>
<section id="a-leap-forward-ai-and-flexible-docking">
<h2>A Leap Forward: AI and Flexible Docking</h2>
<p>As we were particularly interested in the RNA-binding protein Musashi-1 (MSI1) and its interaction with RNA through RNA-binding domains (MSI1-RBDs), we proposed a two-step approach: first, we utilized the AI-based modeling technique Alphafold2 to predict a 3D structure of the MSI1-RBDs based on the amino-acid sequences in humans. Following this, we used a flexible docking technique called PaCS-MD to construct the protein-RNA complexes. Unlike traditional methods that manually ligate RNA to the protein, PaCS-MD automatically constructs the complexes, providing a more natural interaction.</p>
</section>
<section id="validation-and-insights">
<h2>Validation and Insights</h2>
<p>To ensure the validity of the constructed complexes, we evaluated their structural stability using the SIE (Solvation Interaction Energy) method. The results confirmed previous studies, showing that a core set of three nucleotides in MSI1-RBDs plays a significant role in the interaction with the target RNA. Moreover, we identified crucial residues for RNA-binding that aligned well with previous findings.</p>
</section>
<section id="advantages-and-limitations">
<h2>Advantages and Limitations</h2>
<p>Our method offers a significant advantage over conventional techniques. Unlike traditional homology modeling methods, such as Phyre2, our technique captures the essential transitions required for forming protein-RNA complexes. However, we acknowledge that the technique relies on having initial structural information from NMR (Nuclear Magnetic Resonance) coordinates. Without this data, achieving accurate flexible docking can be challenging.</p>
</section>
<section id="future-prospects">
<h2>Future Prospects</h2>
<p>The study provides a compelling demonstration of constructing plausible protein-RNA complexes using a combination of AI-based modeling and flexible docking. This research takes us one step closer to deciphering the complex interactions between proteins and RNA, paving the way for advancements in personalized medicine and drug discovery.</p>
<div class="m-button m-primary">
<a href="../../../../files/papers/Darai-2023.pdf"><span>Download PDF</span></a>
</div>
<aside class="m-frame">
<h3>Abstract</h3>
<p>An efficient structural refinement technique for protein-RNA complexes is proposed based on a combination of AI-based modeling and flexible docking. Specifically, an enhanced sampling method called parallel cascade selection molecular dynamics (PaCS-MD) was extended to include flexible docking to construct protein-RNA complexes from those obtained by AI-based modeling (AlphaFold2). With the present technique, the conformational sampling of flexible RNA regions is accelerated by PaCS-MD, enabling one to construct plausible models for protein-RNA complexes. For demonstration, PaCS-MD constructed several protein-RNA complexes of the RNA-binding Musashi-1 (MSI1) family of proteins, which were validated by comparing a group of crucial residues for RNA-binding with experimental complexes. Our analyses suggest that PaCS-MD improves the quality of complex modeling compared to the standard protocol based on template-based modeling (Phyre2). Furthermore, PaCS-MD could also be a beneficial technique for constructing complexes of non-native RNA-binding to proteins.</p>
</aside>
<section id="citation">
<h3>Citation</h3>
<blockquote>
<p>
<a class="m-flat m-text m-strong" href="https://doi.org/10.1246/bcsj.20230092">A Structural Refinement Technique for Protein-RNA Complexes Using a Combination of AI-based Modeling and Flexible Docking: A Study of Musashi-1 Protein</a><br />
Nitchakan Darai, Kowit Hengphasatporn, Peter Wolschann, <span class="m-text m-ul">Michael T. Wolfinger</span>, Yasuteru Shigeta, Thanyada Rungrotmongkol, Ryuhei Harada<br />
<em>B. Chem. Soc. Jpn.</em> (2023)<br />
</p>
</blockquote>
</section>
<section id="see-also">
<h3>See Also</h3>
<blockquote>
<p>
<a class="m-flat m-text m-strong" href="../../../../blog/2022/Theoretical-studies-on-RNA-recognition-by-Musashi1-RNA-binding-protein/">Theoretical studies on RNA recognition by Musashi 1 RNA–binding protein</a><br />
Nitchakan Darai, Panupong Mahalapbutr, Peter Wolschann, Vannajan Sanghiran Lee, <span class="m-text m-ul">Michael T. Wolﬁnger</span>, Thanyada Rungrotmongkol<br />
<em>Sci. Rep.</em> 12:12137 (2022) | <a class="doi" href="https://doi.org/10.1038/s41598-022-16252-w">doi:10.1038/s41598-022-16252-w</a> | <a class="m-flat m-text" href="../../../../files/papers/Darai-2022.pdf">PDF</a> | <a class="m-flat m-text" href="../../../../files/QuickSlide/QuickSlide__Darai-2022.pdf">Figures</a><br />
</p>
<p>
<a class="m-flat m-text m-strong" href="../../../../blog/2019/Musashi-Binding-Elements-in-Zika-and-Related-Flavivirus-3UTRs-A-Comparative-Study-in-Silico/">Musashi Binding Elements in Zika and Related Flavivirus 3’UTRs: A Comparative Study in Silico</a><br />
Adriano de Bernardi Schneider, <span class="m-text m-ul">Michael T. Wolfinger</span><br />
<em>Sci. Rep.</em> 9(1):6911 (2019) | <a class="doi" href="https://doi.org/10.1038/s41598-019-43390-5">doi:10.1038/s41598-019-43390-5</a> | <a class="m-flat m-text" href="../../../../files/papers/deBernardiSchneider-2019a.pdf">PDF</a> | <a class="m-flat m-text" href="../../../../files/QuickSlide/QuickSlide__deBernardiSchneider-2019a.pdf">Figures</a><br />
</p>
</blockquote>
</section>
</section>
<!-- /content -->
      <footer>
        <p>Posted by Michael T. Wolfinger on <time datetime="2023-06-09T00:00:00+02:00">Fri 09 June 2023</time> in <a href="https://michaelwolfinger.com/blog/category/publications.html">publications</a>. <span class="m-label m-dim">updated <time datetime="2023-06-01T00:00:00+02:00">Thu 01 June 2023</time></span> Tags: <a href="https://michaelwolfinger.com/blog/tag/new-method.html">new method</a>, <a href="https://michaelwolfinger.com/blog/tag/3d.html">3D</a>, <a href="https://michaelwolfinger.com/blog/tag/rna-protein-interaction.html">RNA-Protein interaction</a>, <a href="https://michaelwolfinger.com/blog/tag/ai.html">AI</a>.</p>
      </footer>
    </article>
    <nav class="m-navpanel m-col-m-2">
      <h3>Categories</h3>
      <ol class="m-block-bar-m">
        <li><a href="https://michaelwolfinger.com/blog/category/howto.html">HOWTO</a></li>
        <li><a href="https://michaelwolfinger.com/blog/category/outreach.html">outreach</a></li>
        <li><a href="https://michaelwolfinger.com/blog/category/publications.html">publications</a></li>
      </ol>
      <h3>Tag cloud</h3>
      <ul class="m-tagcloud">
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/3d.html">3D</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/ai.html">AI</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/bacteria.html">bacteria</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/energy-landscapes.html">energy landscapes</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/molecular-epidemiology.html">molecular epidemiology</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/new-method.html">new method</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/ngs.html">NGS</a></li>
        <li class="m-tag-4"><a href="https://michaelwolfinger.com/blog/tag/non-coding-rna.html">non-coding RNA</a></li>
        <li class="m-tag-1"><a href="https://michaelwolfinger.com/blog/tag/novel-viruses.html">novel viruses</a></li>
        <li class="m-tag-4"><a href="https://michaelwolfinger.com/blog/tag/one-health.html">One Health</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/rna-design.html">RNA design</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/rna-kinetics.html">RNA kinetics</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/rna-protein-interaction.html">RNA-Protein interaction</a></li>
        <li class="m-tag-1"><a href="https://michaelwolfinger.com/blog/tag/rna-rna-interaction.html">RNA-RNA interaction</a></li>
        <li class="m-tag-2"><a href="https://michaelwolfinger.com/blog/tag/shape.html">SHAPE</a></li>
        <li class="m-tag-4"><a href="https://michaelwolfinger.com/blog/tag/synthetic-biology.html">synthetic biology</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/tools.html">tools</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/viennarna.html">ViennaRNA</a></li>
        <li class="m-tag-5"><a href="https://michaelwolfinger.com/blog/tag/virus-bioinformatics.html">virus bioinformatics</a></li>
        <li class="m-tag-3"><a href="https://michaelwolfinger.com/blog/tag/xrrna.html">xrRNA</a></li>
      </ul>
    </nav>
  </div>
</div>
</main>
<footer><nav>
  <div class="m-container">
    <div class="m-row">
      <div class="m-col-s-3 m-col-t-6">
        <h3><a href="https://michaelwolfinger.com/">Home</a></h3>
        <ul>
          <li><a href="https://michaelwolfinger.com/research">Research</a></li>
          <li><a href="https://michaelwolfinger.com/publications">Publications</a></li>
          <li><a href="https://michaelwolfinger.com/teaching">Teaching</a></li>
          <li><a href="https://michaelwolfinger.com/team">Team</a></li>
          <li><a href="https://michaelwolfinger.com/blog">Blog</a></li>
        </ul>
      </div>
      <div class="m-col-s-3 m-col-t-6">
        <h3>Services</h3>
        <ul>
          <li><a href="https://michaelwolfinger.com/consulting">Bioinformatics Consulting</a></li>
          <li><a href="https://rnaforecast.com">RNA Forecast</a></li>
          <li><a href="https://xrrna.bio">xrRNA.bio</a></li>
          <li><a href="https://michaelwolfinger.com/contact">Contact</a></li>
          <li><a href="https://michaelwolfinger.com/legal">Legal Note</a></li>
        </ul>
      </div>
      <div class="m-clearfix-t"></div>
      <div class="m-col-s-3 m-col-t-6">
        <h3>My Profiles</h3>
        <ul>
          <li><a href="https://scholar.google.at/citations?user=w0PHGnEAAAAJ&amp;hl=en">Google Scholar</a></li>
          <li><a href="https://www.webofscience.com/wos/author/record/N-9538-2014">Web of Science</a></li>
          <li><a href="https://www.researchgate.net/profile/Michael-Wolfinger">ResearchGate</a></li>
          <li><a href="https://www.scopus.com/authid/detail.uri?authorId=6508361997">Scopus</a></li>
        </ul>
      </div>
      <div class="m-col-s-3 m-col-t-6">
        <h3>Social</h3>
        <ul>
          <li><a href="https://www.linkedin.com/in/michaelwolfinger/">Linkedin</a></li>
          <li><a href="https://twitter.com/mtwolfinger">Twitter</a></li>
          <li><a href="https://github.com/mtw">GitHub</a></li>
          <li><a href="https://orcid.org/0000-0003-0925-5205">ORCID</a></li>
        </ul>
      </div>
    </div>
    <div class="m-row">
      <div class="m-col-l-10 m-push-l-1">
        <p>
        michaelwolfinger.com - Copyright © Michael T. Wolfinger, 2021-2024<br />
        </p>
      </div>
    </div>
  </div>
</nav></footer>
</body>
</html>